scss/elements/_typography.scss

```diff
@@ -79,3 +79,9 @@ hr {
   border-top: none;
   margin: u(1rem 0);
 }
+
+blockquote {
+  border-left: 10px solid $gray-lightest;
+  padding-left: 30px;
+  font-size: u(1.6rem);
```

Comment:
So the blockquote button is actually based inside of the paragraph wagtail component, however, the p tag does not actually wrap the blockquote so it does not inherit the paragraph styles. Which reminds me, I forgot to put in an image of what the blockquote button looks like in wagtail, see below.

![screen shot 2017-03-09 at 9 50 21 am](https://cloud.githubusercontent.com/assets/12799132/23755311/e1e600e2-04ad-11e7-9075-35a33d2b46c0.png)